clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/SCV.scala

```diff
@@ -52,10 +56,13 @@ case class SCV(
   submittedAssembly: Option[String],
   interpretationDescription: Option[String],
   interpretationLastEvaluated: Option[String],
-  interpretationComments: Array[String]
+  interpretationComments: Array[String],
+  vcvTraitSetId: Option[String],
+  rcvId: Option[String]
 )
```

Comment:
If we're fully "flipping the arrows", this should also have fields for `clinicalAssertionTraitSetId` and `clinicalAssertionObservationIds`

Reply:
... But now that the core logic is working, could we possibly undo the partial arrow-flip that's already done?